src/RadForce/test_radiation_force.cpp

```diff
@@ -133,11 +133,12 @@ void RadhydroSimulation<TubeProblem>::setInitialConditionsAtLevel(int lev) {
       state(i, j, k, RadSystem<TubeProblem>::x2RadFlux_index) = 0;
       state(i, j, k, RadSystem<TubeProblem>::x3RadFlux_index) = 0;
 
-      state(i, j, k, RadSystem<TubeProblem>::gasDensity_index) = rho;
       state(i, j, k, RadSystem<TubeProblem>::x1GasMomentum_index) = rho * vel;
       state(i, j, k, RadSystem<TubeProblem>::x2GasMomentum_index) = 0;
       state(i, j, k, RadSystem<TubeProblem>::x3GasMomentum_index) = 0;
+      state(i, j, k, RadSystem<TubeProblem>::gasDensity_index) = rho;
       state(i, j, k, RadSystem<TubeProblem>::gasEnergy_index) = 0;
+      state(i, j, k, RadSystem<TubeProblem>::gasInternalEnergy_index) = -((rho * vel)*(rho * vel))/(2*rho);
```

Comment:
the gas internal energy should not be initialized to be < 0. however, in this case, the problem is isothermal, so it won't affect the solution. here we should just set it to zero.